.cursor/commands/create-test.md

```
@@ -1,276 +1,335 @@
 # Create Test
 
-Create a new Playwright test from a Jira ticket or feature description using the **scenario infrastructure**.
+Create a new Playwright E2E test from a Jira ticket or feature description.
 
 ## Input
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)